piston/utils.py

```diff
@@ -3,6 +3,8 @@
 from django.core.cache import cache
 from django import get_version as django_version
 from decorator import decorator
+import json
```

Comment:
does it matter that we use `simplejson` throughout videobank code, and not `json`?